launcher/src/main/java/org/apache/spark/launcher/ChildProcAppHandle.java

```diff
@@ -46,6 +47,18 @@ public synchronized void disconnect() {
     }
   }
 
+  /**
+   * Since {@link SparkLauncher} runs {@code spark-submit} in a sub-process, its difficult to
```

Comment:
Since this is a public API, good to follow standard javadoc practices like the first sentence being a short description of the method.

Reply:
updated the javadocs